scripts/seed/README.md

```
@@ -0,0 +1,159 @@
+| `route_progress`      | 1 in-progress + 1 completed route                        |
+| `route_completions`   | 1 completed route                                        |
+
+All image URLs point to public Unsplash CDN images.
+
+---
+
+## 4. Adding more mock data
+
+Edit the data files in `scripts/seed/data/`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)